network/src/noise/handshake.rs

```diff
@@ -124,6 +125,8 @@ impl HandshakeAuthMode {
 
 /// The Noise configuration to be used to perform a protocol upgrade on an underlying socket.
 pub struct NoiseUpgrader {
+    /// The validator's own peer id.
+    peer_id: PeerId,
```

Comment:
i've started using `self_peer_id` to make it less confusing with a "remote" peer id

Reply:
I've used `my_` and `their_` also, maybe we should make sure we keep it consistent. No strong opinion :)

Reply:
I've also seen `own_peer_id` along with `self_peer_id`.